portfolio/pom.xml

```diff
@@ -23,6 +23,16 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
```

Comment:
I couldn't find where this is being used, what's it do? (Just curious)

Reply:
It's the App engine dependency to enables the use of Datastore in the project.